op-deployer/pkg/deployer/flags.go

```diff
@@ -13,15 +13,19 @@ import (
 )
 
 const (
-	EnvVarPrefix       = "DEPLOYER"
-	L1RPCURLFlagName   = "l1-rpc-url"
-	CacheDirFlagName   = "cache-dir"
-	L1ChainIDFlagName  = "l1-chain-id"
-	L2ChainIDsFlagName = "l2-chain-ids"
-	WorkdirFlagName    = "workdir"
-	OutdirFlagName     = "outdir"
-	PrivateKeyFlagName = "private-key"
-	IntentTypeFlagName = "intent-type"
+	EnvVarPrefix             = "DEPLOYER"
+	L1RPCURLFlagName         = "l1-rpc-url"
+	CacheDirFlagName         = "cache-dir"
+	L1ChainIDFlagName        = "l1-chain-id"
+	ArtifactsLocatorFlagName = "artifacts-locator"
+	L2ChainIDsFlagName       = "l2-chain-ids"
+	WorkdirFlagName          = "workdir"
+	OutdirFlagName           = "outdir"
+	PrivateKeyFlagName       = "private-key"
+	IntentTypeFlagName       = "intent-type"
+	EtherscanAPIKeyFlagName  = "etherscan-api-key"
```

Comment:
this doesn't need to happen in this PR but we should support alternative escan API URLs as well